substrate/client/service/src/client/wasm_substitutes.rs

```diff
@@ -126,7 +126,7 @@ where
 				let runtime_code = RuntimeCode {
 					code_fetcher: &WrappedRuntimeCode((&code).into()),
 					heap_pages: None,
-					hash: Vec::new(),
+					hash: make_hash(&code),
```

Comment:
Looks like [this](https://github.com/moonbeam-foundation/polkadot-sdk/blob/830fde2a60d60e214d95b3c9da6b6a11bbccf707/substrate/primitives/core/src/traits.rs#L94) doc is outdated?